vcpkg/users/vcpkg-configuration-json.md

```diff
@@ -0,0 +1,143 @@
+---
+title: vcpkg-configuration.json Reference
+description: Reference documentation for the vcpkg-configuration.json file format
+ms.date: 11/30/2022
+---
+# vcpkg-configuration.json Reference
+
+The `vcpkg-configuration.json` file forms part of a project's [manifest](manifests.md), along with [`vcpkg.json`](vcpkg-json.md). All fields in the `vcpkg-configuration.json` file are only used from the top-level project -- the `vcpkg-configuration.json` files in any dependencies are ignored.
+
+In [Manifest Mode](manifests.md), `vcpkg-configuration.json` can be in a separate file beside [`vcpkg.json`](vcpkg-json.md) or it can be embedded in the [`"vcpkg-configuration" field`](vcpkg-json.md#vcpkg-configuration).
+
+In [Classic Mode](classic-mode.md), vcpkg will use the `vcpkg-configuration.json` file in the [root](../commands/common-options.md#vcpkg-root) of the vcpkg instance.
+
+For an overview of using registries with vcpkg, see [Using Registries](registries.md).
+
+## Example
+
+```json
+{
+  "default-registry": {
+    "kind": "git",
+    "repository": "https://internal/mirror/of/github.com/Microsoft/vcpkg",
+    "baseline": "eefee7408133f3a0fef711ef9c6a3677b7e06fd7"
+  },
+  "registries": [
+    {
+      "kind": "git",
+      "repository": "https://github.com/northwindtraders/vcpkg-registry",
+      "baseline": "dacf4de488094a384ca2c202b923ccc097956e0c",
+      "packages": [ "beicode", "beison" ]
+    }
+  ],
+  "overlay-ports": [
+    "./team-ports",
+    "./custom-ports"
+   ],
+  "overlay-triplets": [ "./my-triplets" ]
+}
+```
+This example adds a private registry, `https://github.com/northwindtraders/vcpkg-registry`, as the source for the libraries `beicode` and `beison`. All other ports are found from an internal mirror of the Curated Catalog hosted at `https://internal/mirror/of/github.com/Microsoft/vcpkg`.
```

Comment:
We need to review if we need to make changes to this example repo. Microsoft has a list of pre-approved fictional names, and I can't remember if "Northwind" was on that list. We also need to check if the logo for that used in that repo is an issue. I don't think this review needs to block this PR though, just something to keep in mind for later.